packages/fast-usdc/src/type-guards.js

```diff
@@ -93,3 +93,26 @@ export const PoolMetricsShape = {
   totalRepays: AmountShape,
 };
 harden(PoolMetricsShape);
+
+/** @type {TypedPattern<ChainPolicy>} */
+export const ChainPoliciesShape = M.splitRecord(
+  {
+    nobleContractAddress: EvmHashShape,
+    cctpTokenMessengerAddress: EvmHashShape,
+    confirmations: M.number(),
+    chainId: M.number(),
+  },
+  { chainType: M.number() },
+);
+harden(ChainPoliciesShape);
+
+/** @type {TypedPattern<FeedPolicy>} */
+export const FeedPolicyShape = M.splitRecord(
```

Comment:
I suggest a comment that this shape has to be JSON-happy: no bigint, undefined, remotable, etc.

Reply:
Done.